### PR TITLE
Daemonset for wca

### PR DESCRIPTION
**Pull request by Damenus**

### CODE REVIEW

**Comment by Creatone** [comment]
Because of change project name, rename variables . 
Example:
`RUN pipenv run make owca_package`
to:
`RUN pipenv run make wca_package`

File: manifest/deamonset.yaml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            value: "'1'"
+          - name: CONFIG
+            value: /etc/owca/wca_config.yml
+        resources:
+          requests:
+            cpu: 500m
+            memory: 500Mi
+        volumeMounts:
+          - name: kubelet-certs
+            mountPath: /etc/wca/certs/
```
Comment (ppalucki) [inline manifest/deamonset.yaml]: replace that with volume from secrets

File: manifest/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+    name: wca-config
+data:
+  wca_config.yml: |
+
+    runner: !DetectionRunner
```
Comment (ppalucki) [inline manifest/configmap.yaml]: Replace that with AllocationRunner and use StaticRulesAllocator as default

(again) please use [this ](https://github.com/intel/workload-collocation-agent/blob/kubecon-demo/demo/wca.yaml#L185)as example (you can ignore static rules file)

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+RUN pip install pipenv
+
+WORKDIR /wca
+COPY . .
+
+RUN pipenv install --dev
+RUN pipenv run make wca_package
+
+# Builing final container that consists of wca only.
```
[the remark this answers is not in the document]
Comment (Damenus) [reply Dockerfile]: Done

File: manifest/configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      # Example detection config
+      rdt_enabled: false
+      node: !KubernetesNode
+        # We support two options: cgroupfs, systemd
+        cgroup_driver: cgroupfs
+
+        # Fill needed PATH to key and certificate to access kubelet.
+        client_private_key: "$PATH/client.key"
```
[the remark this answers is not in the document]
Comment (Damenus) [reply manifest/configmap.yaml]: ok, done

File: manifest/deamonset.yaml (hunk 1 above)
Comment (Damenus) [reply manifest/deamonset.yaml]: ok, done

File: manifest/configmap.yaml (hunk 2 above)
Comment (Damenus) [reply manifest/configmap.yaml]: ok, done

File: example/manifest/configmap.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+      action_delay: 5.0
+      rdt_enabled: true
+      enable_derived_metrics: True
+      node: !KubernetesNode
+        # We support two options: cgroupfs, systemd.
+        cgroup_driver: cgroupfs
+        client_private_key: "/etc/wca/certs/client.key"
+        client_cert: "/etc/wca/certs/client.crt"
+        kubelet_endpoint: "https://$OWN_IP_TO_BE_REPLACED:10250"
+        monitored_namespaces: ["wca"]
```
Comment (ppalucki) [inline example/manifest/configmap.yaml]: please use `"default"` as default

Comment (Damenus) [reply example/manifest/configmap.yaml]: done

File: docs/kubernetes.rst [hunk 6]
```diff
@@ -46,3 +46,29 @@ Please refer to `example configuration file for kubernetes <../configs/kubernete
...
+Workload Collocation Agent required private key and certificate to connect with kubelet.
+Example how add this files to Secrets:
+``sudo kubectl create secret generic kubelet-key-crt --from-file=./client.crt --from-file=./client.key --namespace=wca``
+
+3. Add configuration file to ConfigMap
+
+Workload Collocation Agent required configuration file. `Example Allocator as ConfigMap<../manifest/configmap.yaml>`_. To create the resource run
+``kubectl apply -f manifest/configmap.yaml``
+
+4. Run DaemonSet
```
[the remark this answers is not in the document]
Comment (Damenus) [reply docs/kubernetes.rst]: done

File: docs/kubernetes.rst [hunk 7]
```diff
@@ -46,3 +46,29 @@ Please refer to `example configuration file for kubernetes <../configs/kubernete
...
+`Example definition DaemonSet<../manifest/daemonset.yaml>`_.
+Use command ``kubectl apply -f manifest/daemonset.yaml`` to create DaemonSet.
```
[the remark this answers is not in the document]
Comment (Damenus) [reply docs/kubernetes.rst]: done

File: example/manifest/daemonset.yaml [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "9100"
+    spec:
+      terminationGracePeriodSeconds: 10
+      hostPID: true
+      hostIPC: true
+      hostNetwork: true
+
+#      affinity:
```
Comment (Damenus) [inline example/manifest/daemonset.yaml]: I wonder if it can be like this

File: example/manifest/daemonset.yaml [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+#        nodeAffinity:
+#          requiredDuringSchedulingIgnoredDuringExecution:
+#            nodeSelectorTerms:
+#              - matchExpressions:
```
Comment (Damenus) [inline example/manifest/daemonset.yaml]: I wonder if it can be like this

**Pull request merged**